Library/Homebrew/manpages/brew.1.md

```diff
@@ -476,11 +476,13 @@ Note that these flags should only appear after a command.
 
     If `--rebase` is specified then `git pull --rebase` is used.
 
-  * `upgrade [install-options]` [<formulae>]:
+  * `upgrade [install-options] [--cleanup]` [<formulae>]:
     Upgrade outdated, unpinned brews.
 
     Options for the `install` command are also valid here.
 
+    If `--cleanup` is passed, remove existing <formula> version(s) in Cellar.
```

Comment:
I don't know if it's worth mentioning that if you've an old version linked it won't be cleaned up. We may already mention it elsewhere, so possibly redundant.